pkg/api/OWNERS

```diff
@@ -2,3 +2,7 @@ approvers:
 - api-approvers
 reviewers:
 - api-reviewers
+filters:
+  "([^/]+/)?(register|types)\\.go$":
```

Comment:
The original regexes started with `^pkg/api/` but AIUI this takes effect relative to the location of this OWNERS file